packages/@lwc/babel-plugin-component/src/decorators/wire/transform.js

```diff
@@ -168,12 +168,6 @@ function buildWireConfigValue(t, wiredValues) {
                 wireConfig.push(t.objectProperty(t.identifier('method'), t.numericLiteral(1)));
             }
 
-            wireConfig.push(
```

Comment:
Since we need to have the dynamic parameters, we don't need any more `hasParams`.

Note: `hasParams` was added with the idea to remove the `static` and `params` meta on the wire in the future.